docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+
+![Servo board jig](./img/servo_jig.jpeg)
+
+Test steps:
+
+1. Connect jig to PSU using banana leads to the leftmost connections. <!-- TODO add connector colours -->
+2. Set the PSU to 12V and 0.5A.
+3. Connect a USB cable to the board to test.
+4. Slide board into the jig from the bottom.
+5. Rotate the levers on both sides to connect the servos.
+6. Run the script and follow its instructions:<br>
+   `kit_test servo_v4 --fw-ver 4.3 --log servo_testing.csv`
+7. Remove board making a separate pile for boards that failed.
+8. Repeat steps 3-5 and follow the script's instructions for the next board.
+
+### Arduino
+
+Here we are testing that all the pins on the Arduino are functional.
+
+For this test you will need:
+
+- The Arduino test shield
+
+Test steps:
+
+1. Attach the test shield to the Arduino.
+2. Connect a USB cable to the Arduino.
+3. Run the script and follow its instructions:<br>
+   `kit_test arduino --log arduino_testing.csv`
+3. Disconnect the Arduino making a separate pile for Arduinos that failed.
+4. Repeat steps 1-2 and follow the script's instructions for the next Arduino.
+
+
+### Camera
+
+Here we are testing that the camera is accessible and can capture undistorted images.
+
+For this test you will need:
+
+- A printed [test marker](./test-marker.pdf)
+
+!!! note
+
+    For SR2025 we also need to include the `--collect-asset` option since we have not recorded currently recorded all the serial numbers of the cameras.
+
+Test steps:
+
+1. Connect the webcam to the computer
+2. Run the script and follow its instructions:<br>
+   `kit_test camera --log camera_testing.csv`
+3. Disconnect the camera making a separate pile for cameras that failed.
+4. Repeat step 1 and follow the script's instructions for the next camera
+
+### Batteries and Chargers
+
+There are no automated tests for the batteries or chargers.
+However, these items need both testing and collation.
+
+Prior to any testing, the batteries and chargers need to have their asset codes manually scanned so that the inventory can be updated.
+
+Next the IMAX B6 Chargers are used to storage charge the batteries.
+This will be the test for both the battery and the charger.
+To access storage charge follow the [competitor documentation](https://studentrobotics.org/docs/kit/batteries/imax_b6_charger#ChargingChecklist) except when reaching the "LiPo BALANCE" screen continue pressing `Inc.` until the screen reads "LiPo STORAGE".
+
+### USB Cable Testing
+
+As of SR2024, the vast majority of kit issues reported throughout the year and to the helpdesks are due to faulty USB cables.
+For SR2025's Kit Testing we are using a [Treedix USB cable tester](https://www.amazon.co.uk/dp/B0CDP7SJZ1) to try and reduce the number of faulty cables we give out.
+
+## Additional Notes
+
+In order to run all the tests in parallel you would need more power supplies than we have.
+The brain board test can be performed using a power board and battery instead of the power supply.
```

Comment:
```suggestion
The brain board test can be performed using a power board and battery instead of the power supply.
If so, make sure to use a power board which has passed its own testing.
```